infra/docker-compose/docker-compose.batch.yml

```diff
@@ -4,9 +4,9 @@ services:
   batch-serving:
     image: ${FEAST_SERVING_IMAGE}:${FEAST_VERSION}
```

Comment:
Previously, i separated the docker compose file into two because i thought there will be use cases where users would only want to start online serving and not batch serving. Now that i am more familiar with Feast i think that doesn't make much sense any more, especially after the change in this PR, where the basic example assumes that both online and batch serving must be present. Do you think we should have one single docker compose file instead, or keep the current setup?

Reply:
I like the idea of merging. I'll do that.

Reply:
Done.